iis-common/src/test/java/eu/dnetlib/iis/common/spark/TestWithSharedSparkContext.java

```diff
@@ -0,0 +1,50 @@
+package eu.dnetlib.iis.common.spark;
+
+import eu.dnetlib.iis.common.SlowTest;
+import org.apache.spark.SparkConf;
+import org.apache.spark.SparkContext;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.util.Objects;
+
+/**
+ * Support for tests using {@link org.apache.spark.SparkContext} - extends this class to access SparkContext in local mode.
+ */
+@SlowTest
+public class TestWithSharedSparkContext {
+    private static transient SparkContext _sc;
+    private static transient JavaSparkContext _jsc;
```

Comment:
Similar question to the one from other PR, out of curiosity: why do we have to declare those two as transient? Is the test class being serialized?

Reply:
The same [answer](https://github.com/openaire/iis/pull/1189#discussion_r523007224) applies also here, even more because this PR deals with `SparkContext`, not `SparkSession`.